### PR TITLE
[repo] Tidy up Workflow names to make them clearer

### PR DESCRIPTION
**Pull request by andrewnicols**

### CODE REVIEW

File: .github/workflows/lighthouse-report.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Lighthouse Report
+name: "CI - Lighthouse Report"
```
Comment (timhunt) [inline .github/workflows/lighthouse-report.yml:1]: This change does nothing to solve the original problem, which is that if you have never heard of 'Lighthouse' then this name is meaningless.

How about 

"CI - Lighthouse website audit"

It is a pity that workflows can't have a Description: to explain them, unlike some other things in the .yaml file.

**Review by timhunt — COMMENTED** [review]
These are all good improvements, but I think you have failed to solve the originally reported problem. See specific comment.

File: .github/workflows/lighthouse-report.yml (hunk 1 above)
Comment (andrewnicols) [reply .github/workflows/lighthouse-report.yml:1]: I know right! I wish we could have descriptions here

**Comment by andrewnicols** [comment]
I've renamed this workflow per your suggestion @timhunt.

**Review by timhunt — APPROVED** [review]
Looks good now. I think this can be merged.

**Comment by andrewnicols** [comment]
Thanks Tim

**Pull request merged**